website_animate/static/src/scss/o_animate_frontend.scss

```diff
@@ -10,11 +10,12 @@
     backface-visibility: hidden; // avoid flickering
     text-rendering: geometricPrecision; // take care of animated titles
     visibility: hidden;
+}
 
-    &:not(.o_animating) {
```

Comment:
I first tried to do this here but it doesn't work, I don't know why 
`&:not(.o_animating):not(.dropdown-menu .o_animate)`

Reply:
Actually you are doing that in your final PR code? That cannot work :grimacing: Maybe not ready for review yet?

Reply:
Mmh actually it now works: https://caniuse.com/css-not-sel-list
Even if it matches the browsers we support, it is still very new though. And I still consider it a bad practice if it can be avoided, is there no other way?

Reply:
> Actually you are doing that in your final PR code? That cannot work grimacing Maybe not ready for review yet?

Actually my "try" in the code line was using the "&" (to avoid adding code lines) and not multiple component in a not ;)

Reply:
> Mmh actually it now works: https://caniuse.com/css-not-sel-list Even if it matches the browsers we support, it is still very new though. And I still consider it a bad practice if it can be avoided, is there no other way?

yes for sure, I'll ping you when it's done.
I did this only to reduce the number of lines of code.

Reply:
@qsm-odoo Finally, I can't find a solution in CSS. Do you think it's worth doing this in JS?

Reply:
Maybe it will be even more readable in JS indeed (like adding a special class on animated elements in a dropdown and then adapting the css rules accordingly).